Before: b5da6b945112e50e6ec9a7d9185815c058244614f8320e999d4dc49140614ad9
After: f3838f4e7116ec64801ae19321103deb8f58b63008869fdfb744deea4f82503c
Patch: 9dcaa10d553ed3a1c757b2be878852619a2e5f597b153d248ee5c7a588808595
--- test/spec/%plugin_name%_plugin_spec.rb.tt	2012-09-16 14:54:37.000000000 +1000
+++ test/spec/%plugin_name%_plugin_spec.rb.tt	2012-09-11 16:52:34.000000000 +1000
@@ -4,7 +4,7 @@
 require '<%= require_path %>'
 
 describe <%= full_class_name %> do
-  include Corvid::Builtin::ResourcePatchTests
+  include Corvid::ResourcePatchTests
 
   include_resource_patch_tests
 
